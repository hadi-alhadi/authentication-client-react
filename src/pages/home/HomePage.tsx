import { Card, CardTitle } from "react-bootstrap";
import React from "react";
import styled from "styled-components";

const HomePage = () => {
  return (
    <HomePageContainer data-testid="home-page-container">
      <Card style={{ width: "18rem" }}>
        <Card.Body>
          <CardTitle>Welcome to the application.</CardTitle>
        </Card.Body>
      </Card>
    </HomePageContainer>
  );
};
export default HomePage;

const HomePageContainer = styled.div`
  display: flex;
  justify-content: center;
  margin-top: 100px;
`;
