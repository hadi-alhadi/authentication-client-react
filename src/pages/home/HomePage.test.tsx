import React from "react";
import { render, screen } from "@testing-library/react";
import HomePage from "./HomePage";

test("renders home page with welcome message", () => {
  render(<HomePage />);
  expect(screen.getByTestId("home-page-container")).toMatchSnapshot();
});
