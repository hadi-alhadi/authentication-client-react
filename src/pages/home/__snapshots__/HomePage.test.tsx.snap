// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home page with welcome message 1`] = `
<div
  class="sc-beySPh cIGrgw"
  data-testid="home-page-container"
>
  <div
    class="card"
    style="width: 18rem;"
  >
    <div
      class="card-body"
    >
      <div
        class="card-title h5"
      >
        Welcome to the application.
      </div>
    </div>
  </div>
</div>
`;
